blackjax/kernels.py

```diff
@@ -394,6 +392,69 @@ def step_fn(rng_key: PRNGKey, state):
         return SamplingAlgorithm(init_fn, step_fn)
 
 
+class latent_gaussian:
```

Comment:
`marginal_latent_gaussian` ?

Reply:
`marginal_latent_gaussian_sampler`? Since `marginal_latent_gaussian` could be interpreted as INLA like algorithm

Reply:
😩

Reply:
How about `MarginalGradSampler` or `MarginalGradientSampler`?

Reply:
which essentially can be understood as "Marginal gradient sampler for latent Gaussian models"

Reply:
Naming is always the worst :grimacing:

Reply:
> marginal_latent_gaussian

INLA would be called inla though (and would probably live in VI, not MCMC)...